src/error.c

```diff
@@ -334,3 +334,21 @@ void
 raise_invalid_encoding( const char *message ) {
   raise_error( STUMPLESS_INVALID_ENCODING, message, 0, NULL );
 }
+
+int
+stumpless_log_to_error_stream( const char* buffer, int buff_len ) {
+  FILE *error_stream = stumpless_get_error_stream();
+  int ret;
+  bool locked;
+  if( error_stream == NULL ) {
+	return 0;
```

Comment:
Why it returns 0?  Isn't it an error if there is no stream?

Reply:
No, the error stream can be set to `NULL` to suppress error output entirely, so if it is `NULL` then it should simply be ignored.